src/lib.rs

```diff
@@ -186,13 +186,13 @@ impl Parser {
             state => {
                 // Exit action for previous state
                 let exit_action = self.state.exit_action();
-                maybe_action!(exit_action, 0);
+                maybe_action!(exit_action, byte);
 
                 // Transition action
                 maybe_action!(action, byte);
 
                 // Entry action for new state
-                maybe_action!(state.entry_action(), 0);
+                maybe_action!(state.entry_action(), byte);
```

Comment:
Just got a comment, not an issue for documentation purposes about this. I thought this would obviously cause a change in functionality, however there's a good reason why this change is not a problem:

There are only a limited amount of exit and entry actions, these include `Action::None`, `Action::Clear`, `Action::Hook`, `Action::OscStart`, `Action::Unhook` and `Action::OscEnd`.

Looking at the `perform_action` function now, we can see which action even consider the `byte` parameter. The parameter is used by `Action::Print`, `Action::Execute`, `Action::Put`, `Action::OscPut`, `Action::CsiDispatch`, `Action::EscDispatch`, `Action::Collect`, `Action::Param` and `Action::BeginUtf8`.

Comparing these two lists, it becomes clear that the parameter is currently unused entirely when `0` is passed. So passing `byte` as parameter and using it in `Action::Hook`, which previously did not use `byte`, will not create any conflicts.